db/c.cc

```diff
@@ -1070,6 +1070,18 @@ int rocksdb_property_int(
   }
 }
 
+int rocksdb_property_int_cf(
+    rocksdb_t* db,
+    rocksdb_column_family_handle_t* column_family,
+    const char* propname,
+    uint64_t *out_val) {
+  if (db->rep->GetIntProperty(column_family->rep, (propname), out_val)) {
```

Comment:
Should it be `Slice(propname)`?
```suggestion
  if (db->rep->GetIntProperty(column_family->rep, Slice(propname), out_val)) {
```